.meta.toml

```diff
@@ -2,4 +2,7 @@
 # https://github.com/plone/meta/tree/master/config/default
 [meta]
 template = "default"
-commit-id = "05a78883"
+commit-id = "78307cdf"
+
+[dependencies]
+ignores = "['plone.rfc822']"
```

Comment:
Why is this in the ignores?
It is in install_requires, so this line looks not needed. But I am not behind my laptop, so I can't try it.

Reply:
Not in my laptop either, but I do remember 😃

It is due to being a soft dependency, there is an extra for it. It that is actually true, or not, I did not spend that much time looking into it

Reply:
Seems fine.